apps/tsilhqotin-language-hub/src/Components/Pages/Contact/Contact.tsx

```diff
@@ -0,0 +1,57 @@
+import DevicesIcon from '@mui/icons-material/Devices';
+import LocalPhoneIcon from '@mui/icons-material/LocalPhone';
+import PlaceIcon from '@mui/icons-material/Place';
+import { Divider } from '@mui/material';
+import ScrollToTop from '../../Widgets/ScrollButton/ScrollToTop';
+import './Contact.css';
+import { ContactsDisplay } from './ContactsDisplay';
+import getContactInfos from './getContactInfos';
+
+const contactInfosReadResult = getContactInfos();
+
+export function Contact() {
+    if (contactInfosReadResult instanceof Error)
+        return (
+            <div>
+                <h1>Error Boundary</h1>
+                <p>
+                    This application has encountered the following error:
+                    {contactInfosReadResult.message}
+                </p>
+            </div>
+        );
+    return (
+        <div className="page">
```

Comment:
BrThe Contact (maybe `ContactContainer` or `ContactPage`) component is just a container that fetches the state (from the config) and flows it as props to the presenter. Put all of this JSX into `ContactsDisplay` for better separation of concerns.